addons/stock_account/models/product.py

```diff
@@ -7,6 +7,13 @@
 from odoo.exceptions import ValidationError
 from collections import defaultdict
 
+STOCK_PROPERTY_FIELDS = [
+    'property_stock_account_input_categ_id',
+    'property_stock_account_output_categ_id',
+    'property_stock_valuation_account_id',
+    'property_stock_account_production_cost_id',
+]
+
```

Comment:
Not sure why this is defined outside of the class, as it's not used anywhere else.
Also, it contains an account that should be defined in `mrp_account` imho. What I'd suggest would be to have a method that gives this list in the class, and then override it in `mrp_account` to add the production account.